egui/src/layout.rs

```diff
@@ -158,7 +158,7 @@ impl Layout {
             main_wrap: false,
             main_align: Align::Center, // looks best to e.g. center text within a button
             main_justify: false,
-            cross_align: Align::Center,
+            cross_align: Align::LEFT,
```

Comment:
This is confusing - the _cross_ axis is the vertical axis in this case, so `Align::TOP` would be a better description here.